src/java.base/share/classes/java/lang/classfile/ClassFile.java

```diff
@@ -1478,6 +1478,9 @@ default List<VerifyError> verify(Path path) throws IOException {
     /** 67 */
     int JAVA_23_VERSION = 67;
 
+    /** 68 */
```

Comment:
We need `@since 24` here.

Reply:
Ah, good catch; looks like I was treating that as an internal API element when it is indeed a public API element. I've filed JDK-8330458 to fix-up the Java 23 constant as well.